meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -526,9 +526,20 @@ if [ "$EFI" = "1" ]; then
         EFIPART_LABEL=balena-efi
     fi
 
+    # remove duplicate entries
+    for label in resinOS "${BOOT_ENTRY_LABEL}"; do
+        duplicates="$(efibootmgr \
+		| grep "${label}" \
+		| sed 's/Boot*//g' \
+		| sed "s/* ${label}//g")"
+        for entry in ${duplicates}; do
+            efibootmgr -B -b "${entry}"
+	done
+    done
+
```

Comment:
Why not just leave the managing of the UEFI environment to the UEFi firmware?

Reply:
These entries are persistent, even if the EFI binary is gone, the entry remains. This line removes duplicates, ensuring that only the most recently created entry remains. Again, this is a consolidation of the existing approach in balena-intel.

Reply:
ack